content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,156 @@
+
+3. **Create Your Repository**:
+   - Click the ‘Use this template’ button to generate your own copy of the repository.
+
+   <img src="images/template.png" alt="Use Template Repository" width="550"/>
+<!-- ![Use Template Repository](images/template.png) -->
+
+## Enable GitHub Actions
+GitHub Actions allow automated tasks to run. They are enabled by default, but we need to enable them to publish the results
+1. **Go to Repository Settings**:
+   - Access your repository’s settings page.
+
+2. **Configure Build and Deploy Source**:
+   - Navigate to the “Pages” section and set the build and deploy source to GitHub Actions.
+
+3. **Allow Actions**:
+   - Ensure that actions are enabled to create the results page.
+
+   <img src="images/gitpages.png" alt="Enable Git Actions" width="550"/>
+<!-- ![Enable GitHub Actions](images/gitpages.png) -->
+##  Edit the `info.yaml` File
+
+1. **Adjust Wokwi ID**:
+   - For Wokwi projects, update the Wokwi ID in the `info.yaml` file to match the ID from your Wokwi project.
+   - For HDL projects, use the source files section as covered in a separate guide titled [Working with an HDL]().
+
+   <img src="images/WOKWI_Project_ID.png" alt="Wokwi ID" width="550"/>
+<!-- ![Set the Wokwi ID](images/WOKWI_Project_ID.png) -->
+
+2. **Fill Out Metadata**:
+   - Enter the title, author information, and your Discord username if applicable.
+3. **Describe Pins**:
+   - Provide descriptions for the pins to aid in testing other designs. 
+
+   <img src="images/info_yaml.png" alt="yamlinfo" width="550"/>
+
+    - Save your changes by clicking the commit button.
+
+    <img src="images/commitinfo.png" alt="commitinfo" width="550"/>
+<!-- 
+![info.yaml](images/info_yaml.png)  -->
+
+<!-- ![Commit the modified info.yaml](images/commitinfo.png) -->
+##  Edit the `info.md` File in the Docs Directory
+
+  **Update Documentation**:
+   - Include detailed instructions on how to experiment with your design. This documentation is useful for users receiving the chip and for you to recall design details later. Note that leaving the info.md unmodified will cause the 'docs' action to fail. Save and commit your changes. Commit can be done in similar way as in Step 2.
+
+   <img src="images/documentation.png" alt="doc" width="550"/>
+<!-- ![Documentation](images/documentation.png) -->
+
+
+## Wokwi Actions
+
+1. **Access Actions Tab**:
+   - Go to the ‘Actions’ tab in your repository to view results. Note that each commit will trigger these actions to run again. Find out more: how the GitHub actions work to build the GDS (link to zero to asic GDS terminology page) files for your project, you can check here (link to /making_asics).
+    
+2. **Actions for Wokwi**:
+
+    - docs: This action generates a preview of how your documentation will look.
+    - fpga:
+    - gds: Action to generate the gds for your project, results for warnings, utilization statistics, and cell usage.
+    - wokwi test:
```

Comment:
If you created a truth table for your wokwi design, this action will test your design with the truth table.